<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLight</class>
 <widget class="QDialog" name="EditLight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="light">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pos_z">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="radius_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Intensity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_light">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Light</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_g">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pos_y">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="color_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_r">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pos_x">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="col_b">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="position">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="intensity">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <double>0.300000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="position_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Smooth Shadow:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="nb_light">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="radius_label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Number of lights :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dist_light">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="radius_label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Distance of lights :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
